tools/isobuild/meteor-npm.js

```diff
@@ -1203,3 +1227,30 @@ var logUpdateDependencies = function (packageName, npmDependencies) {
   runLog.log(packageName + ': updating npm dependencies -- ' +
              Object.keys(npmDependencies).join(', ') + '...');
 };
+
+function removeFileDependencies(tree, npmTree) {
+  function removeInModule(module, moduleNpmTree) {
+    var newModule = { version: module.version };
+
+    if (module.dependencies) {
+      _.each(module.dependencies, function (subModule, name) {
+        subModuleNpmTree = moduleNpmTree?.dependencies?.[name];
+        if (subModuleNpmTree?.version.startsWith("file:"))
+          return;
+        if (!newModule.dependencies)
+          newModule.dependencies = {};
+        newModule.dependencies[name] = removeInModule(subModule, subModuleNpmTree);
+      });
+    }
+    return newModule;
+  }
+
+  var newTopLevelDependencies = {};
+  _.each(tree.dependencies, function (module, name) {
+    moduleNpmTree = npmTree?.dependencies?.[name];
+    if (moduleNpmTree?.version.startsWith("file:"))
+      return;
+    newTopLevelDependencies[name] = removeInModule(module, moduleNpmTree);
+  });
+  return {dependencies: newTopLevelDependencies};
```

Comment:
Let's also add missing spaces around `{}` here also.

Reply:
It doesn't seem that meteor uses this standard?

Reply:
It's inconsistent, I can add it to my change I guess.